File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/pattern1/p2.c
#include<stdio.h>
void main(){
	int rows;


	printf("Enter the rows: ");
	scanf("%d",&rows);

	for(int i=1;i<=rows;i++){

		char ch = 'a';
		int x=1;
		for(int j=1;j<=3;j++){
			if(i==1 || i==3){

			printf(" %d ",x);
			x++;
			
			}else{

				printf(" %c",ch);
				ch++;
			}
		
		}
		printf("\n");
	}
}

File: java-main/java-main/JavaDSA_Super-X/Assi_6/p2.java
import java.util.Scanner;

class Demo{
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter the number of rows: ");
        int rows = scanner.nextInt();
        printPattern(rows);
        scanner.close();
    }

static void printPattern(int rows) {
        int startValue = 1;

        for (int i = 1; i <= rows; i++) {
            int currentValue = startValue;
            for (int j = 1; j <= i; j++) {
                System.out.print(currentValue + " ");
                currentValue += i + j;
            }
            startValue = (i + 1) * (i + 1);
            System.out.println();
        }
    }
}

